frontend/src/pages/UpdateSettings.js

```diff
@@ -33,13 +35,17 @@ export default function UpdateSettings() {
   const handleSubmit = event => {
     event.preventDefault()
     updateSettings(newSettings, token).catch(error => console.error(error))
+    setUpdate(true)
```

Comment:
Danger! only call `setUpdate` when the request was successful - put it inside a then call.
Maybe rename it from ùpdate` to `redirectToHome` or something?